integration-test/background/click-to-load-youtube.js

```diff
@@ -244,6 +256,145 @@ describe('Test YouTube Click To Load', () => {
 
         await page.close()
     })
+
+    it('CTL: YouTube Preview', async () => {
+        // Open the test page and start logging network requests.
+        const page = await browser.newPage()
+        const pageRequests = []
+        const clearRequests = await logPageRequests(page, pageRequests)
+
+        // Navigate to test page
+        await pageWait.forGoto(page, testSite)
+
+        // Once the user clicks to preview a video, should change from blocked state
+        // to preview state and request only YouTube Preview images
+        clearRequests()
```

Comment:
I don't think clearing the requests here is correct. We want to know if the test page manages to sneak any requests through during load.